scripts/ci/images-v2/Dockerfile.ci-spack-el8-base

```diff
@@ -1,4 +1,5 @@
-FROM almalinux/almalinux:8
+ARG BASE_IMAGE=almalinux/almalinux:8
+FROM $BASE_IMAGE
```

Comment:
Good catch,

For the CUDA images we build this docker file with something like `sudo docker ... --build-arg BASE_IMAGE=nvidia-base-image-rocky8`. That is why we need to parametrize this

Reply:
> I might need to update the tests to check the automatic detection (right now I set the memory space to CUDA manually), but for now this will be fine.

So far this build does not run the ADIOS2 tests, the reason is that we do not have a machine with a GPU yet. In the coming months we expect to provide such machine

Reply:
Ok, so I have a few months to make sure the tests are covering everything :)